wren-ai-service/src/pipelines/generation/chart_adjustment.py

```diff
@@ -93,6 +93,10 @@ def run(
             generation_result = orjson.loads(replies[0])
             reasoning = generation_result.get("reasoning", "")
             if chart_schema := generation_result.get("chart_schema", {}):
+                # sometimes the chart_schema is still in string format
```

Comment:
Im curious about what is the string format be like. could you provide me an example? thanks

Reply:
have discussed this comment with Jimmy offline